tests/frameworks/OSXShims/include/IwMalloc.h

```diff
@@ -0,0 +1,67 @@
+//******************************************************************************
+//
+// Copyright (c) 2015 Microsoft Corporation. All rights reserved.
+//
+// This code is licensed under the MIT License (MIT).
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+//
+//******************************************************************************
+
+#ifndef __IWMALLOC_H
+#define __IWMALLOC_H
+
+#include <string.h>
+
+#ifdef _WIN32
```

Comment:
>_WIN32 [](start = 7, length = 6)

This seems like a weird choice for a def to key off of. I believe WIN32 is also defined and is a little less private / compiler dependent. Additionally, CF already uses TARGET_OS_XXX for platform differences. Have you considered using that instead?